### PR TITLE
New documentation

### PR DESCRIPTION
**Pull request by dokato**
# New documentation

I completely changed the way README is built. Now you don't need any external packages, you do it straight from Knit, and then you use pkgdown to build static html website with docs.

Images of components changed to static graphic as rendering caused to many problems especially with `Rmd -> md -> (html) pkgdown` pipeline.

I fixed also some inconsistencies in CHANGELOG and added blue-ish template for docs.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -1,138 +1,98 @@
...
 [![codecov](https://codecov.io/gh/Appsilon/shiny.semantic/branch/master/graph/badge.svg)](https://codecov.io/gh/Appsilon/shiny.semantic)
-</div>
 
-<div class="section level2">
+[![cranlogs](https://cranlogs.r-pkg.org/badges/shiny.semantic)](https://CRAN.R-project.org/package=shiny.semantic)
+
+[![total](https://cranlogs.r-pkg.org/badges/grand-total/shiny.semantic)](https://www.rpackages.io/package/shiny.semantic)
+<!-- badges: end -->
+
+
```
Comment (jakubnowicki) [inline README.Rmd:37]: Can we set badges in one line?

File: README.Rmd [hunk 2]
```diff
@@ -1,138 +1,98 @@
...
- 
-```{r, echo = FALSE, message = FALSE, fig.height = 1, fig.width = 5.5}
-create_example(before_ui_demo(), width="100%", height="90%")$htmlwidget
-```
- 
-</div>
-<br>
-<div class="column">
+
+![Before](man/figures/ss_before.png)
```
Comment (jakubnowicki) [inline README.Rmd:44]: Not working: 
image

File: README.Rmd [hunk 3]
```diff
@@ -1,138 +1,98 @@
...
- 
-</div>
-</div>
-</div>
-
-<!-- #Basic tutorial article is available on [Appsilon Data Science blog](your_future_art_link). -->
-<!-- Live demo link below -->
-<!-- TODO Analogy to http://shiny.rstudio.com/gallery/widget-gallery.html -->
+
+![After](man/figures/ss_after.png)
```
Comment (jakubnowicki) [inline README.Rmd:48]: Not working:
image

File: README.Rmd [hunk 4]
```diff
@@ -1,138 +1,98 @@
...
+
+Component examples
+------------------
+
+![Components](man/figures/semantic_components.png)
```
Comment (jakubnowicki) [inline README.Rmd:53]: Quality of this image is quite low, it does not look good.

File: README.Rmd [hunk 5]
```diff
@@ -211,116 +159,36 @@ shinyApp(ui = ui(), server = server)
...
-    # Your UI code
-    )
-  )
+shinyUI(semanticPage(
+title = "My page",
+tags$script(jsCode),
+suppressDependencies("bootstrap"),
+# Your UI code
+)
+)
```
Comment (jakubnowicki) [inline README.Rmd:174]: Please add indentation to the code.

File: README.Rmd (hunk 1 above)
Comment (dokato) [reply README.Rmd:37]: It was like that at first, but then it looks quite ugly in docs/index.html for some reason.

File: README.Rmd (hunk 2 above)
Comment (dokato) [reply README.Rmd:44]: This is weird. It works for me when I knit it locally. I think that github looks for relative paths only on master, so as long as we don't merge these in, it wouldn't display

File: README.Rmd (hunk 4 above)
Comment (dokato) [reply README.Rmd:53]: Ok, TODO

File: README.Rmd (hunk 5 above)
Comment (dokato) [reply README.Rmd:174]: Sure, TODO

**Pull request merged**